.github/workflows/go.coverage.yml

```diff
@@ -22,6 +22,7 @@ jobs:
 
       - name: Test With Coverage
         run: |
+          go install github.com/fatih/faillint@latest || true
```

Comment:
`|| true` would ignore install errors, which is probably not what we want.

```suggestion
          go install github.com/fatih/faillint@latest
```

Reply:
ok 👍 fixed